commands/milestone_report_command.go

```diff
@@ -0,0 +1,61 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+	"github.com/hanochg/piperika/http"
+	"github.com/hanochg/piperika/report"
+	"github.com/hanochg/piperika/utils"
+	"github.com/jfrog/jfrog-cli-core/plugins/components"
+	"sync"
+	"time"
+)
+
+const defaultBranchName string = "master"
+
+var services = []string{"access", "metadata", "event", "router", "jfconnect", "mission_control", "integration"}
+
+func GetMilestoneReport() components.Command {
+	return components.Command{
+		Name:        "milestone_report",
+		Description: "Get milestone report for Infra QA blender",
```

Comment:
This code is public, change the naming to be more generic like "report" and "Get reports of something"

Reply:
I did some changes in names and description. Tell me if it's good enough.